docs/http.rst

```diff
@@ -35,8 +35,8 @@ The following section describes how remote servers can send Push
 Notifications to apps running on remote User Agents.
 
 API methods requiring Authorization must provide the Authorization
-header containing the authrorization token. The Authorization token is returned
-as "secret" in the registration response.
+header containing the authorization secret. The Authorization secret is
```

Comment:
What do you think of calling it the "registration secret", to make it clearer that it's only for the bridge API? "Authorization secret" feels a little generic.